README.rst

```diff
@@ -13,45 +13,36 @@ Beamline Simulation using `sirepo-bluesky`_ and Synchrotron Radiation Workshop (
 
 Purpose
 -------
-
-Use machine learning to simulate beamlines similar to SRW with lower computational costs.
-The goal is to determine whether or not a neural network can construct a beamline simulation given data from sirepo-bluesky.
+Use machine learning techniques to simulate beamlines given data from SRW.
 
 Installation
 ------------
 
 - Install `VirtualBox`_ on your computer.
 - Install `Vagrant`_ using the terminal.
 - Git clone this repository. Then use ``cd deep-beamline-simulation`` to move into the top level directory of this repository.
-- You will see a Vagrantfile containing virtual machine setup information. Start the virtual machine using ``vagrant up`` followed by ``vagrant ssh``.
-- Once the virtual machine is running, finish the Miniconda installation,
-.. code:: bash
+- You will see a Vagrantfile containing virtual machine setup information. Start the virtual machine using ``vagrant up`` followed by ``vagrant ssh``. To reload the virtual machine use ``vagrant reload`` and to update the machine for any changes use ``vagrant provision``.
 
-   bash Miniconda3-latest-Linux-x86_64.sh
-
-- Close and reload the VM. Create a conda environment, 
-.. code:: bash
+- It is recommended to check the status of Mongo DB using ``sudo systemctl status mongod``. If the status is 'dead' use ``sudo systemctl start mongod.service`` to start running Mongo DB.
```

Comment:
Here's another place where I would like some example output such as this (or exactly this):
```
vagrant@vagrant:~$ sudo systemctl status mongod
● mongod.service - MongoDB Database Server
     Loaded: loaded (/lib/systemd/system/mongod.service; enabled; vendor preset: enabled)
     Active: inactive (dead) since Mon 2021-12-20 14:13:47 UTC; 3s ago
       Docs: https://docs.mongodb.org/manual
    Process: 647 ExecStart=/usr/bin/mongod --config /etc/mongod.conf (code=exited, status=0/SUCCESS)
   Main PID: 647 (code=exited, status=0/SUCCESS)

Dec 20 14:07:41 vagrant systemd[1]: Started MongoDB Database Server.
Dec 20 14:13:47 vagrant systemd[1]: Stopping MongoDB Database Server...
Dec 20 14:13:47 vagrant systemd[1]: mongod.service: Succeeded.
Dec 20 14:13:47 vagrant systemd[1]: Stopped MongoDB Database Server.
vagrant@vagrant:~$ sudo systemctl start mongod
vagrant@vagrant:~$ sudo systemctl status mongod
● mongod.service - MongoDB Database Server
     Loaded: loaded (/lib/systemd/system/mongod.service; enabled; vendor preset: enabled)
     Active: active (running) since Mon 2021-12-20 14:13:57 UTC; 1s ago
       Docs: https://docs.mongodb.org/manual
   Main PID: 2655 (mongod)
     Memory: 156.0M
     CGroup: /system.slice/mongod.service
             └─2655 /usr/bin/mongod --config /etc/mongod.conf

Dec 20 14:13:57 vagrant systemd[1]: Started MongoDB Database Server.```